aws-traffic-mirroring/templates/seekret-sniffer.yaml

```diff
@@ -33,10 +33,10 @@ Parameters:
   BucketSecretKey:
     Type: String
     Description: Secret key for the bucket
-  BucketFolderName:
+  Workspace:
```

Comment:
change the documentation of the traffic mirroring

Reply:
wasn't mentioned in the readme at the first place